ui/cryptomaterial/slider.go

```diff
@@ -57,8 +57,12 @@ func (s *Slider) Layout(gtx C, items []layout.Widget) D {
 		s.items = items
 		s.isSliderItemsSet = true
 	}
-	s.handleClickEvent()
 
+	if len(s.items) == 0 {
+		return D{}
+	}
```

Comment:
i see no harm in showing the asset slider for a watch only wallet

Reply:
or is a watch only wallet's balance not calculated as part of the slider?

if it's not then ignore this

Reply:
> or is a watch only wallet's balance not calculated as part of the slider?
> 
> if it's not then ignore this

Exactly, calculate total assets balance not include watch-only wallet, additional it causes crash app